mitsurunner.yaml

```diff
@@ -93,13 +90,12 @@ sensor:
 # State machine to trigger defrosting when needed
   - platform: template
     id: state_machine
-    name: "Defrost state machine"
+    name: "Temperature delta"
```

Comment:
Do not rename this. Will make more difficult to understand. 
Instead: 
Put this to report the delta MQTT message by adding row:
    state_topic: $topic_delta

Reply:
Resolved this by accident